CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 - Added environment setting for agent_trigger_method (#2025)
 - Expose compile data as exported by `inmanta compile --export-compile-data` via API (inmanta/inmanta-telco#54)
 
+## Upgrade notes
+- DynamicProxy.\_\_getattr\_\_ now raises an AttributeError instead of a NotFoundException when an attribute can not be found, for compatibility with Python's builtin `hasattr` (#2991)
```

Comment:
Why the backslashes? Put it between backticks? The note also does not mention that this change is backwards compatible.

Reply:
Good point, I'll add it.